### DISCUSSION TITLE
How to inject provided object inside Android Service?

### DISCUSSION THREAD

**Discussion by ryccoatika** [evant/kotlin-inject#270] [Q&A] [answered]
I have a Firebase Messaging Service, used to listen every new token up coming.
So I have to inject repository inside my own service which extended from FirebaseMessagingService()

So the question is, how I inject the repository into my Firebase Messaging Service class regarding this class will be placed inside Android Manifest

**Comment by evant** [comment, marked as answer]
https://github.com/evant/kotlin-inject/blob/main/docs/android.md#injecting-into-platform-classes

Docs talk about Activity but it should also apply to Service

**Reply by ryccoatika** [reply]
yes, solved.
I realized that service has onCreate method.

thanks
